modules/ROOT/pages/functions/temporal/duration.adoc

```diff
@@ -346,6 +345,7 @@ duration.inMonths(instant1, instant2)
 | If `instant1` has a time zone component and `instant2` does not, the time zone component of `instant2` is assumed to be the same as that of `instant1`, and vice versa.
 | If `instant1` has a date component and `instant2` does not, the date component of `instant2` is assumed to be the same as that of `instant1`, and vice versa.
 | Any difference smaller than a whole month is disregarded.
+| To get the actual number of months as an `INTEGER`, call the months component of the returned `DURATION` object via `.months`, see see xref:values-and-types/temporal.adoc#cypher-temporal-accessing-components-durations[Components of durations].
```

Comment:
```suggestion
| Call the months component of the returned `DURATION` object via `.months` to get duration total number of months, see xref:values-and-types/temporal.adoc#cypher-temporal-accessing-components-durations[Components of durations].
```
For clarity and readability and to follow the tone of the rest of the section (ie avoid using _you_).